src/r2mm/data/ThunderstorePackages.ts

```diff
@@ -36,6 +38,14 @@ export default class ThunderstorePackages {
             map.set(pkg.getFullName(), pkg);
             return map;
         }, new Map<String, ThunderstoreMod>());
+
+        ThunderstorePackages.LATEST_VERSIONS = ThunderstorePackages.PACKAGES.reduce((map, pkg) => {
```

Comment:
Calculating this now seems unnecessary use of resources?

Reply:
Oops, good catch